cpp/src/aztec/stdlib/primitives/bigfield/bigfield.test.cpp

```diff
@@ -859,7 +860,11 @@ template <typename Composer> class stdlib_bigfield : public testing::Test {
 };
 
 // Define types for which the above tests will be constructed.
-typedef testing::Types<plonk::UltraComposer, plonk::TurboComposer, plonk::StandardComposer, honk::StandardHonkComposer>
+typedef testing::Types<plonk::UltraComposer,
```

Comment:
When is the right time to fully hook up `StandardPlonkComposer` in place of `StandardComposer`?

Reply:
I guess, when we split everything, so that we can start getting rid of oldschool composers